ocaml/sdk-gen/csharp/templates/XenServer.csproj.mustache

```diff
@@ -7,11 +7,11 @@
     <GenerateAssemblyInfo>True</GenerateAssemblyInfo>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <Title>XenServer.NET</Title>
-    <Authors>Citrix Systems, Inc.</Authors>
+    <Authors>Cloud Software Group, Inc.</Authors>
     <PackageId>$(AssemblyName).NET</PackageId>
     <Product>$(AssemblyName).NET</Product>
     <Description>.NET wrapper for the XenServer API</Description>
-    <Copyright>Copyright (c) Citrix Systems, Inc. All rights reserved.</Copyright>
+    <Copyright>Copyright (c) 2007-2023 Cloud Software Group, Inc. All Rights Reserved.</Copyright>
     <PackageTags>citrix hypervisor virtualization sdk jsonrpc .net c# xen xenserver</PackageTags>
     <PackageLicenseExpression>BSD-2-Clause</PackageLicenseExpression>

```

Comment:
Are we not updating the license of the generated SDKs to LGPLv2, too?

Reply:
No need to.